fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/PullUpPredicates.java

```diff
@@ -325,8 +343,8 @@ private ImmutableSet<Expression> getAvailableExpressions(Set<Expression> predica
         return ImmutableSet.copyOf(newPredicates);
     }
 
-    private boolean hasAgg(Expression expression) {
-        return expression.anyMatch(AggregateFunction.class::isInstance);
+    private boolean supportPullUpAgg(Expression expr) {
```

Comment:
Does it mean the original hasAgg judgement actually contains any unexpected pulled up agg cases, after this restriction?